controls/srg_gpos/SRG-OS-000377-GPOS-00162.yml

```diff
@@ -6,6 +6,7 @@ controls:
             (PIV) credentials.
         rules:
             - sssd_certificate_verification
+            - var_sssd_certificate_verification_digest_function=sha1
             - install_smartcard_packages
             - dconf_gnome_enable_smartcard_auth
```

Comment:
```suggestion
```

Please remove this rule from all the SRGs. No RHEL8 profiles use this rule and this is probably not applicable to RHEL9.

There are three occurrences `SRG-OS-000375-GPOS-00160,SRG-OS-000376-GPOS-00161,SRG-OS-000377-GPOS-00162`